karma.config.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env node */
+
+const path = require(`path`);
+
+process.env.CHROME_BIN = require(`puppeteer`).executablePath();
+process.env.FIREFOX_BIN = require(`playwright`).firefox.executablePath();
+const SAUCE_LAB = parseInt(process.env.SAUCE_LAB, 10) === 1;
+
+// Check out https://saucelabs.com/platforms for all browser/platform combos
+const sauceLaunchers = {
+  sl_chrome_latest: {
+    base: `SauceLabs`,
+    browserName: `chrome`,
+    browserVersion: `latest`,
+    platformName: `macOS 12`,
+    'sauce:options': {
+      extendedDebugging: true,
+    },
+  },
+  sl_chrome_old: {
+    base: `SauceLabs`,
+    browserName: `chrome`,
+    browserVersion: `76`,
```

Comment:
how about we move this up to 90
```suggestion
    browserVersion: `90`,
```